src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -135,18 +135,26 @@ def consume_captions(caption_list: list) -> dict:
     for caption in caption_list:
         if 'caption_id' not in caption:
             raise ValueError('A caption ID must be set for each caption')
+
+		# Setting the default background color to white if there is no background color passed in
+        foreground_color = 'white'
+		# Setting the default test to be not underlined if no underline variable is passed in
+        underlined = False
+        if 'foreground_color' in caption and 'color' in caption['foreground_color']:
+            caption_color = caption['foreground_color']['color']
+			
+        if 'underline' in caption:
+            underlined = caption['underline']
+        
+        caption_metadata['caption_string'] += utils.create_byte_pairs_for_midrow_style(
```

Comment:
We can probably ditch using a dictionary for storing just a list of bytes, instead just use a list called something like `caption_bytes`

Reply:
Outside the scope of this ticket. There could be other testing needed to do that.